generator/main.ts

```diff
@@ -32,7 +32,7 @@ async function readFile(path: string) {
 async function main(idls: Array<string>) {
   for (let idl of idls) {
     let parsedData = webidl.parse(await readFile(idl));
-    console.log(nunjucks.render('./template/example.tpl', {
+    console.log(nunjucks.render('./template/example.njk', {
         context: parsedData[0]
     }));
   }
```

Comment:
Let's make this function return error code.
If this is normal exit, we can return 0 at end of this function.